build/action.yml

```diff
@@ -0,0 +1,34 @@
+name: 'Build Python package'
+description: 'Build a Python package and store the results'
+inputs:
+  artifact-name:
+    default: "artifact"
+  artifact-path:
+    default: "dist"
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install poetry
+      run: |
+        curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -
```

Comment:
[According to the documentation](https://python-poetry.org/docs/#osx--linux--bashonwindows-install-instructions), this has been changed to `get-poetry.py`.

Reply:
The documentation says that `get-poetry.py` will be replaced by `install-poetry.py`


> The get-poetry.py script described [here](https://python-poetry.org/docs/master/#installation) will be replaced in Poetry 1.2 by install-poetry.py. From Poetry 1.1.7 onwards, you can already use this script as described here.

In the master branch they recommend installing poetry using the following command, so maybe is the best approach:
`curl -sSL https://install.python-poetry.org | python3 -`

Reply:
My mistake!